apps/common-app/src/examples/RuntimeTests/ReJest/TestRunner/UpdatesContainer.ts

```diff
@@ -82,16 +83,18 @@ export function createUpdatesContainer() {
 
   function pushLayoutAnimationUpdates(tag: number, update: Record<string, unknown>) {
     'worklet';
-    if (global._IS_FABRIC) {
-      // layout animation doesn't work on Fabric yet
-      return;
-    }
+
     // Deep Copy, works with nested objects, but doesn't copy functions (which should be fine here)
     const updatesCopy = JSON.parse(JSON.stringify(update));
     if ('backgroundColor' in updatesCopy) {
       updatesCopy.backgroundColor = convertDecimalColor(updatesCopy.backgroundColor);
     }
-    _updateNativeSnapshot([{ tag, update }], jsUpdates.value.length - 1);
+    if (!global._IS_FABRIC) {
+      // TODO Implement native snapshots for layout animations on Fabric
+      _updateNativeSnapshot([{ tag, update }], jsUpdates.value.length - 1);
+    } else {
+      brokenNativeSnapshotsOnFabric.value = true;
```

Comment:
This would have some negative side effects, for example it would broke this test:

https://github.com/software-mansion/react-native-reanimated/blob/ddabf41dc178236daadb6be34d9032622f74797b/apps/common-app/src/examples/RuntimeTests/tests/animations/withDecay/basic.test.tsx#L46

Cause only in case of layout animations the native snapshots are unimplemented, so we can assume that if function `notifyAboutProgress` has never been called, then native snapshots work fine on fabric.